src/command_modules/azure-cli-acr/HISTORY.rst

```diff
@@ -7,7 +7,7 @@ Release History
 ++++++
 * Allow force push a helm chart that doesn't exist.
 * Allow runtime operations without ARM requests.
-* Deprecating '--resource-group' parameter from 'az acr login', 'az acr repository', and 'az acr helm' commands.
+* Deprecating '--resource-group' and add '--suffix' parameter for 'az acr login', 'az acr repository', and 'az acr helm' commands.
```

Comment:
I don't understand. @djyou, doesn't look like you actually deprecated resource group? The commands still expose it right?

Reply:
Please ignore, I didn't notice that the deprecating comment was from a previous pr.